app/upgrades/v17/upgrades.go

```diff
@@ -21,13 +26,23 @@ import (
 	poolmanagertypes "github.com/osmosis-labs/osmosis/v17/x/poolmanager/types"
 )
 
+const (
+	mainnetChainID = "osmosis-1"
+	e2eChainA      = "osmo-test-a"
+	e2eChainB      = "osmo-test-b"
+)
+
+var notEnoughLiquidityForSwapErr = errorsmod.Wrapf(gammtypes.ErrInvalidMathApprox, "token amount must be positive")
```

Comment:
nit: why don't we import this error from the place where it is returned in superfluid?

Reply:
There is no error to import. Its from gamm, and every instance of this is manually entered. Example:

https://github.com/osmosis-labs/osmosis/blob/ecbb38361499f48ea775280d9a73af1a3e022424/x/gamm/keeper/swap.go#L61-L62
https://github.com/osmosis-labs/osmosis/blob/ecbb38361499f48ea775280d9a73af1a3e022424/x/gamm/pool-models/balancer/pool.go#L519-L520
https://github.com/osmosis-labs/osmosis/blob/ecbb38361499f48ea775280d9a73af1a3e022424/x/gamm/pool-models/balancer/pool.go#L576-L577